src/Symfony/Component/Form/AbstractExtension.php

```diff
@@ -136,7 +136,7 @@ protected function loadTypeExtensions()
     /**
      * Registers the type guesser.
      *
-     * @return FormTypeGuesserInterface|null A type guesser
+     * @return FormTypeGuesserInterface|null A type guesser or null
```

Comment:
maybe drop `A type guesser or null` completely? :smile: Seems quite clear already

Reply:
Yes done!

Reply:
you haven't done it.